docs/api/rest.md

```diff
@@ -499,7 +499,7 @@ curl -X POST -H "Authorization: Bearer ABCDEFGH" \
 
 #### POST /api/events/&lt;event_type>
 
-Fires an event with event_type
+Fires an event with event_type. Please be mindful of the data structure as mentioned in the [Data Docs](https://data.home-assistant.io/docs/events/#database-table).
```

Comment:
"Data Docs" sounds weird.
Shouldn't it say, "as documented on our Data Science portal"?

Reply:
That does sound better :) thank you!